timeseries-aggregator/src/main/scala/com/expedia/www/haystack/trends/aggregation/WindowedMetric.scala

```diff
@@ -52,6 +53,8 @@ class WindowedMetric private(var windowedMetricsMap: mutable.TreeMap[TimeWindow,
     * @param incomingMetricData - incoming metric data
     */
   def compute(incomingMetricData: MetricData): Unit = {
+    timeInTopicMetricPointHistogram.update(incomingMetricData.getTimestamp() - System.currentTimeMillis())
```

Comment:
Is incomingMetricData.getTimestamp() is in micros?

Reply:
It looks like the MetricDataTransformer object converts the [span's microseconds](https://github.com/ExpediaDotCom/haystack-idl/blob/master/proto/span.proto#L33) to seconds 
https://github.com/ExpediaDotCom/haystack-trends/blob/master/span-timeseries-transformer/src/main/scala/com/expedia/www/haystack/trends/transformer/MetricDataTransformer.scala#L41

Reply:
I'll update my PR with that info